modules/homefinder/widgets/class-homes-for-sale-widget.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * HomeFinder Homes for Sale Widget (http://www.homefinder.com/widgets/)
+ *
+ * @package RE-PRO
+ */
+
+	/* Exit if accessed directly. */
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->